tests/pykafka/test_balancedconsumer.py

```diff
@@ -4,11 +4,10 @@
 import unittest2
```

Comment:
```test_simpleconsumer``` already has a test for EARLIEST and LATEST offsets.  Just thought I'd add one for balanced consumer as well.